style.css

```diff
@@ -0,0 +1,158 @@
+:root {
+  --banner-bg-color: rgba(255, 255, 255, 0.301);
+}
+@font-face {
+  font-family: "ultraregular";
+  src: url("/fonts/ultra.woff") format("woff2"), url("/fonts/ultra.woff") format("woff");
+  font-weight: normal;
+  font-style: normal;
+}
+/***nav-bar***/
+.nav {
+  box-shadow: var(--nav-bar-shadow);
+}
+.nav-heading {
+  text-decoration: none;
+  color: var(--primary-color);
+  font-family: "ultraregular", sans-serif;
+  font-size: 1.5rem;
+}
+/*******
+/*** banner ***/
+.banner {
+  height: calc(100vh - 5.5rem);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 2rem;
+  position: relative;
+  background-image: url(/assets/bg-img3.jpg);
+  background-repeat: no-repeat;
+  background-position: top center;
+  background-size: cover;
+}
+.banner-content {
+  padding: 2rem;
+  border-radius: 0.3rem;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  row-gap: 1rem;
+  position: absolute;
+  background-color: var(--sidebar-fallback-bg-color);
+}
+@supports (backdrop-filter: blur(3px)) {
+  .banner-content {
+    background-color: var(--banner-bg-color);
+    backdrop-filter: blur(3px);
+  }
+}
+.banner-heading {
+  font-family: "ultraregular", sans-serif;
+  font-size: 3rem;
+  margin: auto;
+  color: var(--primary-color);
+  word-spacing: 20px;
+}
+.banner-sub-heading {
+  font-size: 1.3rem;
+}
+.banner-cta {
+  display: flex;
+  column-gap: 1rem;
+  row-gap: 1rem;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+/*******
+
+/*** category-section ***/
+.section-heading {
+  font-size: 2.5rem;
+}
+.title-hr {
+  border: 2px solid var(--primary-color);
+  background-color: var(--primary-color);
+  width: 70px;
+  height: 0;
+}
+.category-grid {
+  padding: 3rem 2rem;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 0.5rem;
+  justify-items: center;
+  align-items: center;
+}
+.card {
+  width: fit-content;
+}
+.overlay-text {
+  font-size: 1.2rem;
+  padding: 0.5rem;
+  border-radius: 0.3rem;
```

Comment:
don't use REM for  border radius if it  is less than 8px, as it will shift-shape based on pixel density.